package.json

```diff
@@ -1,11 +1,11 @@
 {
   "private": true,
   "scripts": {
-    "postinstall": "node script/get-quasar.js",
+    "update-quasar": "node bower_components/purescript-quasar-test-starter/quasar/get-quasar.js",
```

Comment:
yuck

Reply:
Yeah, but it's better than maintaining the script in multiple places and having to update it every time we change something 😕

Reply:
Hopefully my Rust script will be a better solution ^^